buku

```diff
@@ -4852,7 +4876,7 @@ def prompt(obj, results, noninteractive=False, deep=False, listtags=False, sugge
                 if index < 0 or index >= count:
                     print('No matching index %s' % nav)
                     continue
-                browse(results[index + cur_index][1])
+                browse(results[index][1])
```

Comment:
This matches up with the earlier change (`cur = next_index`)

Also the check just above clearly expects `index` to be between 1 and `count` :sweat_smile: